db/migrate/20171030175029_add_ancrestry_to_content_nodes.rb

```diff
@@ -0,0 +1,11 @@
+class AddAncrestryToContentNodes < ActiveRecord::Migration[5.1]
+  def up 
+    add_column :content_nodes, :ancestry, :string
+    add_index :content_nodes, :ancestry
```

Comment:
Keep it for now, but in the future, you can do this in one line:

```ruby
add_column :content_nodes, :ancestry, :string, index: true
```

Reply:
Ah yeah

Reply:
Typo: Ancrestry should be Ancestry in file and class name